Doc/whatsnew/3.11.rst

```diff
@@ -348,26 +369,31 @@ For example::
         cls.__ne__ = ...
         return cls
 
-    # The create_model decorator can now be used to create new model
-    # classes, like this:
+    # The create_model decorator can now be used to create new model classes:
     @create_model
     class CustomerModel:
         id: int
         name: str
 
-    c = CustomerModel(id=327, name="John Smith")
+    c = CustomerModel(id=327, name="Eric Idle")
 
 See :pep:`681` for more details.
 
 (Contributed by Jelle Zijlstra in :gh:`91860`. PEP written by
 Erik De Bonte and Eric Traut.)
 
-PEP 563 May Not Be the Future
+
+.. _whatsnew311-pep563-deferred:
+
+PEP 563 may not be the future
 -----------------------------
 
-* :pep:`563` Postponed Evaluation of Annotations, ``__future__.annotations``
-  that was planned for this release has been indefinitely postponed.
-  See `this message <https://mail.python.org/archives/list/python-dev@python.org/message/VIZEBX5EYMSYIJNDBF6DMUMZOCWHARSO/>`_ for more information.
+:pep:`563` Postponed Evaluation of Annotations
+(the ``from __future__ import annotations`` :ref:`future statement <future>`)
+that was originally planned for release in Python 3.10 has been deferred again.
```

Comment:
I prefer the current phrasing of "indefinitely postponed" rather than "has been deferred again"; it feels like it describes the situation better.

Reply:
Yes, "deferred again" suggests that PEP 563 will become the default in 3.12, which isn't looking likely to me.

Reply:
I restored original "indefinitely" wording, only retaining "deferred" instead of "postponed". The reason I changed "postponed" to begin with was due to word repetition and the significant potential for confusion due to the title and subject of the PEP in question being "postponed" evaluation of annotations (it even confused me for a moment).